docs/reference/mapping/types/core-types.asciidoc

```diff
@@ -379,7 +379,9 @@ defaults to `true` or to the parent `object` type setting.
 |`ignore_malformed` |Ignored a malformed number. Defaults to `false`.
 
 |`numeric_resolution` |The unit to use when passed in a numeric values. Possible
```

Comment:
Thought about this, but as this can still be returned in the mapping data, not finding the option at all  here might be more confusing for the user